src/Converters/PollyConverter.php

```diff
@@ -17,11 +20,20 @@ class PollyConverter implements Converter
      */
     protected $client;
 
+    /**
+     * Construct converter.
+     *
+     * @param array $config
+     */
     public function __construct($config)
     {
         $credentials = $this->getCredentials($config['credentials']);
 
-        $this->client = new \Aws\Polly\PollyClient(['version' => $config['version'], 'credentials' => $credentials, 'region' => $config['region']]);
```

Comment:
I import all the necessary AWS class. I'm not sure if this will work even the AWS package isn't installed by default in this package. I'll check what it looks like in my local setup.